src/Blocks/BlockOre.h

```diff
@@ -55,6 +55,11 @@ class cBlockOreHandler :
 				a_Pickups.push_back(cItem(E_ITEM_NETHER_QUARTZ));
 				break;
 			}
+			case E_BLOCK_CLAY:
+			{
+				a_Pickups.push_back(cItem(E_ITEM_CLAY,4));
```

Comment:
I made the space.

On Thu, Apr 23, 2015 at 7:49 PM, Mattes D <notifications@github.com> wrote:

> In src/Blocks/BlockOre.h
> <https://github.com/mc-server/MCServer/pull/1891#discussion_r28985935>:
>
> > @@ -55,6 +55,11 @@ class cBlockOreHandler :
> >  				a_Pickups.push_back(cItem(E_ITEM_NETHER_QUARTZ));
> >  				break;
> >  			}
> > +			case E_BLOCK_CLAY:
> > +			{
> > +				a_Pickups.push_back(cItem(E_ITEM_CLAY,4));
>
> Needs a space after the comma (basic typesetting rules)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mc-server/MCServer/pull/1891/files#r28985935>.
>